index.bs

```diff
@@ -988,16 +988,11 @@ To verify the consistency of two versions of a register, given that total-entrie
 
 # Minting a new entry # {#minting}
 
-  To mint a new entry in the register:
-  * POST a new item in [[#json-representation]] to the ..mint url.. with the proposed new entry number.
-  * The item will first appear as a new item in the store
-  * <em> .. status codes and error cases </em>
-  * <em> .. 202 Accepted with a Location header of the [[#entry-resource]] for the new entry</em>
-  * The proposer can only trust it has been secured when the entry
-    is listed in the [[#entries-resource]] with the same [[#entry-number-field]],
-    and the [[#entry-resource]] is a covered by one or more [[#digital-proofs]] for the register.
-
-The item MUST NOT contain empty fields.
```

Comment:
Is this part an implementation detail of minting or does it say something about what you can store in a register? Assuming the former but just want to check.

Reply:
All that should be derived from the requirements for the different interfaces (i.e. entry resource, digital proofs resources).